packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -680,6 +681,18 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
   );
   loggerWithEventDetails.debug(`Available slots: ${JSON.stringify(computedAvailableSlots)}`);
 
+  // We only want to run this on single targeted events and not dynamic
+  if (!computedAvailableSlots.slots && input.usernameList?.length === 1) {
+    await handleNotificationWhenNoSlots({
```

Comment:
This could potentially delay the response for slots. Not sure by how significant that would be in production.

Worst case, if upstash goes down our availabilities go down.

Reply:
Good shout - we can wrap this in a try catch I think that’s possible the best way to solve this

Reply:
Ideally this is the type of stuff we put in a background job but we don’t have a way to do this rn

Reply:
I don’t think the removing the await is reliable especially in server less right? 

I’ll deffo implement the try catch